### PR TITLE
Blog on how boolean works in Ruby

### PR DESCRIPTION
**Pull request by utsav00**
## What did you do?
- Wrote a blog on the lack of boolean in Ruby lang
- [x] Add a picture - @swathynair19 can you help me out here please?

## Why did you do it?
- Thought it'd be interesting to learn about why a high level language does not have a boolean class

### CODE REVIEW

**Comment by akshayvadher** [comment]
Interesting writing style. Looks good to me.

**Review by 2KAbhishek — APPROVED** [review]
A very fun read!

nit: Would be good to run prettier once on the file, I noticed some minor formatting inconsistency

File: content/blog/ruby-boolean.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+irb> (false.methods & true.methods).count
+59
+irb> (false.methods & true.methods) - Object.methods
+=> [:&, :|, :^]
+
+# Most definitely
+
+```
+
+> They only have three methods precisely in common with each other (╭ರ_•́)
```
Comment (2KAbhishek) [inline content/blog/ruby-boolean.md:95]: (╭ರ_•́)

Maybe we can use the monocles emoji here 🧐 some people may get confused by this

Comment (utsav00) [reply content/blog/ruby-boolean.md:120]: Didn't want to add a hint of yellow or color in a blog, so went with ascii emojis

**Pull request merged**